src/sentry/api/endpoints/project_rule_preview.py

```diff
@@ -68,5 +69,10 @@ def post(self, request: Request, project) -> Response:
             on_results=lambda x: serialize(x, request.user),
             count_hits=True,
         )
+
+        inbox_details = get_inbox_details([Group(id=int(g["id"])) for g in response.data])
```

Comment:
https://github.com/getsentry/sentry/blob/ece6d6c751166e1fe05d6b187f207179f1e8cd8f/src/sentry/models/groupinbox.py#L126

it looks like `get_inbox_details` just pulls the ids out of the `Group` object anyways. assuming there's no consistency issues, can we just make `get_inbox_details` a wrapper for a method that takes in a list of `id`? that way we don't have to hit the db here

Reply:
sounds good